agent/xds/testdata/routes/connect-proxy-with-chain-and-router.golden

```diff
@@ -29,7 +29,12 @@
             },
             {
               "match": {
```

Comment:
I wonder if we need golden files for a range of envoy feature variants now. :thinking: